tests/Nox.Types.Tests/Types/HashedText/HashedTextTests.cs

```diff
@@ -1,11 +1,74 @@
 ﻿// ReSharper disable once CheckNamespace
+using System.Security.Cryptography;
+
 namespace Nox.Types.Tests.Types;
 
 public class HashedTextTests
 {
+
+    [Fact]
+    public void Nox_HashedText_Constructor_WithoutOptions_ReturnsHashedValue()
```

Comment:
follow the convention for naimng the tests
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#best-practices
```suggestion
    public void HashedText_Constructor_WithoutOptions_ReturnsHashedValue()
```